### PR TITLE
add deimosmuc to map

### PR DESCRIPTION
**Pull request by deimosmuc**

### CODE REVIEW

**Comment by githubteacher** [comment]
@deimosmuc You have a great start here! One quick note, you will need to format the file exactly like it is shown in the instructions in order for it to work:

screen-shot-2016-10-11-at-10 01 58-am

Please @ mention me when you finish and I will approve the changes!

File: _pins/deimosmuc.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+githubHandle: Deimosmuc
```
Comment (githubteacher) [inline _pins/deimosmuc.json]: @deimosmuc please be sure to add a line at the beginning and end of the file with `---`!

**Comment by lizcobbe** [comment]
Adding motivation. Yay!

File: _pins/deimosmuc.json (hunk 1 above)
Comment (deimosmuc) [inline _pins/deimosmuc.json]: done :)

**Pull request merged**